### PR TITLE
upgrade core to 3.15.29

### PR DESCRIPTION
**Pull request by nigeriabuddy**
**Affects:** \<api-version-number>.

---

### CODE REVIEW

File: src/main/java/com/capitalone/dashboard/service/AutoDiscoveryServiceImpl.java [hunk 1]
```diff
@@ -44,6 +44,7 @@ public AutoDiscoveryServiceImpl(AutoDiscoveryRepository autoDiscoveryRepository,
     @Override
     public AutoDiscovery save(AutoDiscoveryRemoteRequest request) throws HygieiaException {
         String autoDiscoveryId = request.getAutoDiscoveryId();
+        request.getMetaData().getProperties();
```
Comment (tatlax3636) [inline src/main/java/com/capitalone/dashboard/service/AutoDiscoveryServiceImpl.java:47]: Is this info ever used?

Comment (nireeshT) [reply src/main/java/com/capitalone/dashboard/service/AutoDiscoveryServiceImpl.java:47]: this line of code is not required. Please remove it

Comment (nigeriabuddy) [reply src/main/java/com/capitalone/dashboard/service/AutoDiscoveryServiceImpl.java]: It has been removed now

**Review by tatlax3636 — APPROVED** [review]
LGTM

**Pull request merged**